source/replication.rst

```diff
@@ -0,0 +1,142 @@
+
+The cross-site replication involves configuring one Percona XtraDB Cluster as *Source*, and another Percona XtraDB Cluster as *Replica* to allow an asynchronous replication between them:
+
+ .. image:: ./assets/images/pxc-replication.svg
+   :align: center
+
+The Operator automates configuration of *Source* and *Replica* Percona XtraDB Clusters, but the feature itself is not bound to Kubernetes. Either *Source* or *Replica* can run outside of Kubernetes, be regular MySQL and be out of the Operators’ control. 
+
+This feature can be useful in several cases: for example, it can simplify migration from on-premises to the cloud with replication, and it can be really helpful in case of the disaster recovery too.
+
+.. note:: Cross-site replication is based on `Automatic Asynchronous Replication Connection Failover<https://dev.mysql.com/doc/refman/8.0/en/replication-asynchronous-connection-failover.html>`_. Therefore it requires  MySQL 8.0.22+ (Percona XtraDB Cluster 8.0.22+) to work.
+
+.. Describe how to stop/start replication
+   Describe how to perform a failover
+
+Setting up MySQL for asynchronous replication without the Operator is described `here <https://www.percona.com/blog/2021/04/14/what-you-can-do-with-auto-failover-and-percona-distribution-for-mysql-8-0-x/>`_ and is out of the scope for this document.
+
+Configuring the cross-site replication for the cluster controlled by the Operator is explained in the following subsections.
+
+.. contents:: :local:
+
+.. _operator-replication-source:
+
+Configuring cross-site replication on Source instances
+------------------------------------------------------
+
+You can configure *Source* instances for cross-site replication with ``spec.pxc.replicationChannels`` subsection in the ``deploy/cr.yaml`` configuration file. It is an array of channels, and you should provide the following keys for the channel in your *Source* Percona XtraDB Cluster:
+
+* ``pxc.replicationChannels.[].name`` key is the name of the channel,
+
+* ``pxc.replicationChannels.[].isSource`` key should be set to ``true``.
+
+Here is an example:
+
+.. code:: yaml
+
+   spec:
+     pxc:
+       replicationChannels:
+       - name: pxc1_to_pxc2
+         isSource: true
+
+The cluster will be ready for asynchronous replication when you apply changes as usual:
+
+.. code:: bash
+
+   $ kubectl apply -f deploy/cr.yaml
+
+.. _operator-replication-expose:
+
+Exposing instances of Percona XtraDB Cluster
+--------------------------------------------
+
+You need to expose every Percona XtraDB Cluster Pod of the *Source* cluster to
+make it possible for the *Replica* cluster to connect. This is done through the
+``pxc.expose`` section in the ``deploy/cr.yaml`` configuration file as follows.
+
+.. code:: yaml
+
+   spec:
+     pxc:
+       expose:
+         enabled: true
+         type: LoadBalancer
+         loadBalancerSourceRanges:
+           - 10.0.0.0/8
+         annotations: 
+           networking.gke.io/load-balancer-type: "Internal"
```

Comment:
we say below " Please take into account, that you will need this IP address to be external
   for cross-region replication in most cases."

why do we have internal load balancer in the example? :)